client/main.js

```diff
@@ -18,10 +18,10 @@ var socket = io.connect('http://' + location.host, {
 // instantiate the updater of the view
 new StatusUpdater(socket, util.elm('title'), util.elm('banner'), util.elm('browsers'));
 window.karma = new Karma(socket, util.elm('context'), window.open,
-	window.navigator, window.location);
+    window.navigator, window.location);
 
-// TODO(julie): is there a better way to store this?
```

Comment:
I don't think we're going to think of a better way of allowing the test to find the socket.  For instance, vojta's old way was super hacky.  This is at least explicit.